frontend/src/components/dashboard/Dashboard.js

```diff
@@ -254,11 +254,10 @@ export default function Dashboard() {
                   <DescLinks name="Description Links"/>
                 </Container>
               </Route>
-              <Route exact path="/picture-id">
-                <Container className={classes.withPadding}>
-                  <ArtworkCloseUpCard/>
-                </Container>
-              </Route>
+              <Route
+                exact path="/gallery/:id"
+                component={ArtworkCloseUpCard}
```

Comment:
By using the component prop of Route now the `id` will pass as a prop to ArtworkCloseUpCard

Reply:
Just a clarification on how this works, does the ":id" part of the path get replaced with the actual id of the artwork?

Reply:
[This](https://scotch.io/courses/using-react-router-4/route-params) article cleared it up for me.  Very cool!

Reply:
That's a nice article!